### ISSUE TITLE
Specify class directly next to node

### ISSUE THREAD

**Issue by notramo** [knsv/mermaid#835]
Class definition is too far from node:
```mermaid
A --> B[node text]

classDef myClass fill:#0f0,stroke:#333,stroke-width:3px;
classDef anotherClass fill:#ff0,stroke:#333,stroke-width:3px;

class B anotherClass
class A myClass
```
I think it would be easier to understand:
```mermaid
A.myClass --> B[node text].anotherClass

classDef myClass fill:#0f0,stroke:#333,stroke-width:3px;
classDef anotherClass fill:#ff0,stroke:#333,stroke-width:3px;
```

**Comment by knsv** [comment]
The syntax with a single dot is intuitive but unfortunately breaks backwards compatibility in flowcharts as for instance this graph would break:

```mermaid
graph TB
   176.123.34.56 --> 172.123.35.78
```

Instead I propose we adapt another delimiter, instead of '.' we can use ':::'

**Comment by knsv** [comment]
Released in 8.2.6
